### PR TITLE
removing the metrics cmdlets as they are not working,

### PR DESCRIPTION
**Pull request by NaveenGoli**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NaveenGoli__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ngoli). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by AzureRT** [comment]
Can one of the admins verify this patch?

File: src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj [hunk 1]
```diff
@@ -125,7 +125,6 @@
     <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Cmdlets\GetAzureWebAppMetrics.cs" />
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj]: Please remove the files as well

File: src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj [hunk 2]
```diff
@@ -134,7 +133,6 @@
     <Compile Include="Cmdlets\StartAzureWebApp.cs" />
     <Compile Include="Cmdlets\StopAzureWebApp.cs" />
     <Compile Include="Cmdlets\AppServicePlan\GetAzureAppServicePlan.cs" />
-    <Compile Include="Cmdlets\AppServicePlan\GetAzureAppServicePlanMetrics.cs" />
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj]: Please remove the files as well

Comment (NaveenGoli) [inline src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj]: we will re enable them as we are working on a fix in backend. we just wanted to ensure these cmdlets are not enabled. Hence not planning to remove the files.

Comment (yadavbdev) [inline src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj]: You can remove those files now and later just revert the commit to bring them back :)

**Pull request merged**